docs/runtime-contract.md

```diff
@@ -173,9 +175,7 @@ autoscaled container is available to serve that request.
 
 ### Inbound network connectivity
 
-Inbound network connectivity is assumed to use HTTP/1.1 compatible transport, to
-enable the runtime environment to determine whether a container is "busy" or not
-for purposes of scaling CPU and removing idle containers.
+Inbound network connectivity is assumed to use HTTP/1.1 compatible transport.
```

Comment:
Should we permit 1.1 or 2.0?

Reply:
I kept it same as what it already is in the current RTC. If we want to change it to HTTP/2.0, that can be a separate conversation and possibly a separate PR?

Reply:
Note that we are only supporting the `h2c` flavor of HTTP/2, not the ALPN (TLS negotiation) version, because that requires developers to manage certificate material within their container. I think our assumption is that in most cases, `h2c` will occur via `Upgrade` header on an HTTP/1.1 connection, but there is also the prior knowledge form, so "HTTP/1.1 compatible transport" might not be appropriate.

In any case, the change here is basically removing the rationale for why the transport is assumed, which doesn't change the contract.